src/HaskellCI.hs

```diff
@@ -860,6 +854,13 @@ genTravisFromConfigs argv opts isCabalProject config prj@Project { prjPackages =
             , not (null l)
             ]
 
+        unless (null (cfgLocalGhcOptions config)) $ forM_ pkgs $ \Pkg{pkgName} -> do
```

Comment:
I guess it's irrelevant whether `cfgRawProject` or `cfgLocalGhcOptions` comes first?

Reply:
I actually, now. I think `rawProject` should came last, so you could do

```
local-ghc-options: -O2
raw-project:
  package tricky
    ghc-options: -O
```

i.e. you could still override "all packages" stuff in `raw-project`

Reply:
I've swapped the order of `local-ghc-options` and `raw-project`.